pkg/cmd/docgen/funcs.go

```diff
@@ -24,6 +24,7 @@ import (
 	"sort"
 	"strings"
 
+	markdown "github.com/golang-commonmark/markdown"
```

Comment:
What's up with the alias here?

Reply:
Hm not sure what happened. Will clean up.